core/src/main/java/org/infinispan/stats/impl/CacheContainerStatsImpl.java

```diff
@@ -64,7 +64,11 @@ public void resetStatistics() {
       if (getStatisticsEnabled()) {
          for (String cn : cm.getCacheNames()) {
             if (cm.cacheExists(cn)) {
-               cm.getCache(cn).getAdvancedCache().getStats().reset();
+               try {
+                  cm.getCache(cn).getAdvancedCache().getStats().reset();
+               } catch (SecurityException e) {
+                  // no-op, skipping this cache
```

Comment:
It's definitely a problem if you're trying to read/reset the statistics of one cache and Infinispan reads/resets the statistics of all caches. The stack trace in the JIRA description doesn't seem right, so I can't tell what attribute the CLI was trying to read, maybe you can add some more information about reproducing this?

My thinking was that these are container-level attributes/operations, so whoever tries to use them should be a global ADMIN. That means it's ok to fail if the user is not a global ADMIN, even if we have a single secure cache and the user is an ADMIN on that cache. (However, the cache-level ADMIN should be able to read/reset the statistics on that particular cache.) 

We can do this by checking the permissions once, before we start iterating on the caches, and then using a `PrivilegedAction` to skip the security checks of the individual caches.

I'm not sure how this is exposed in the CLI, so let me know if any of this make sense @vjuranek @tristantarrant !